docassemble/ALAffidavitOfIndigency/data/questions/affidavit_of_indigency_supplement_body.yml

```diff
@@ -0,0 +1,1015 @@
+    datatype: yesnoradio
+---
+code: |
+  if not has_household_members:
+    household.there_are_any = False
+---
+question: |
+  How many of your household members are employed?
+subquestion: |
+  A household member is someone you live and share expenses with.
+fields:
+  - no label: household.target_number
+    datatype: integer
+---
+code: |
+  if not has_household_members:
+    household.target_number = 0
+---
+sets:
+  - household[i].name.first
+  - household[i].name.last
+question: |
+  What is the name of your ${ ordinal(i) } household member?
+fields:
+  code: household[i].name_fields()
+---
+id: household member job number, name or no name
+question: |
+  How many jobs does ${ household[i].name } have?
+fields:
+  - Number of jobs: household[i].jobs.target_number
+    datatype: integer
+---
+id: household member job name
+question: |
+  What is the job title of ${ household[i].name }'s ${ ordinal(j) } job?
+fields:
+  - Job title: household[i].jobs[j].source
+---
+question: |
+  Do you have another household member that is employed?
+subquestion: |
+  You have already told us about ${ comma_and_list(household.complete_elements()) }.
+fields:
+  - no label: household.there_is_another
+    datatype: yesnoradio
+---
+code: |
+  def full_household_income():
+    total_from_user = users[0].jobs.net_total(times_per_year=12) + users[0].nonemployment.total(times_per_year=12)
+    total_from_others = sum([per.jobs.net_total() for per in household])
+    return total_from_user + total_from_others
+---
+code: |
+  def list_household(all_hh):
+      member_list = []
+      for member in all_hh:
+          if len(member.jobs) > 0:
+              all_jobs = comma_list([f"a {mem_job.source} for {mem_job.employer.name} ({mem_job.employer.address.on_one_line()})" for mem_job in member.jobs])
+              member_list.append(f"{member.name}, works as {all_jobs}, making {currency(member.jobs.net_total(times_per_year=12))}")
+      return comma_list(member_list, comma_string=",\n", and_string=",\n")
+---
+id: how many jobs do you have
+question: |
+  How many jobs do you have?
+fields:
+  - no label: users[0].jobs.target_number
+    datatype: integer
+---
+id: any non-employment incomes
+question: |
+  Do you have any of the below sources of income, not from employment?
+fields:
+  - no label: users[0].nonemployment.selected_types
+    datatype: checkboxes
+    code: |
+      nonemployment_income_terms_ordered
+---
+code: |
+  users[0].nonemployment.selected_types
+  users[0].nonemployment.move_checks_to_list(None, nonemployment_income_terms)
+  users[0].nonemployment.moved = True
+---
+code: |
+  users[0].nonemployment.moved
+  for income_type in users[0].nonemployment.elements:
+    income_type.complete
+  users[0].nonemployment.revisit = True
+  users[0].nonemployment.gathered = True
+---
+question: |
+  Tell us about your ${ ordinal(i) } income
+subquestion: |
+  For example, you might recieve disability benefits (SSDI) for $1000 once a month.
```

Comment:
```suggestion
```
This is confusing when it comes up in the context of another specific income type than SSDI.